src/main/java/com/spartronics4915/frc2020/commands/LowerPanelRotatorCommand.java

```diff
@@ -0,0 +1,39 @@
+package com.spartronics4915.frc2020.commands;
+
+import com.spartronics4915.frc2020.subsystems.PanelRotator;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class LowerPanelRotatorCommand extends CommandBase {
+
+    private PanelRotator mPanelRotator;
+    // You should only use one subsystem per command. If multiple are needed, use a CommandGroup.
+    public LowerPanelRotatorCommand(PanelRotator mSubsystem) {
+      mPanelRotator = mSubsystem;
+    }
+
+    // Called when the command is initially scheduled.
+    @Override
+    public void initialize() {
+    }
+
+    // Called every time the scheduler runs while the command is scheduled.
+    @Override
+    public void execute() {
+        mPanelRotator.lower();
+
+    }
+
+    // Returns true when the command should end.
+    @Override
+    public boolean isFinished() {
```

Comment:
Once we get the robot, we can test this.

I know that Dana expressed interest in having the arm stay up even when run into the chute, but that might not be necessary - it depends on the mechanic's design.